tests/Meilisearch.Tests/KeyTests.cs

```diff
@@ -105,7 +105,7 @@ public async Task DeleteOneKey()
                 Description = "Key to delete document to all indexes.",
                 Actions = new string[] { "documents.delete" },
                 Indexes = new string[] { "*" },
-                ExpiresAt = null,
+                ExpiresAt = DateTime.Parse("2042-04-02T00:42:42Z"),
```

Comment:
I think this test doesn’t have to be changed, since the field support dates and null types, right?

Reply:
Ah, I've reverted that test.